elyra/metadata/schemas/airflow.json

```diff
@@ -47,15 +48,17 @@
           "format": "uri",
           "default": "https://api.github.com",
           "uihints": {
-            "category": "Apache Airflow"
+            "category": "Apache Airflow",
+            "placeholder": "https://api.github.com"
```

Comment:
Although the placeholder value will never be visible when a default value also exists, it seems like the placeholder should be like the others nevertheless  (e.g., `"https://your-github-endpoint"`).

Reply:
I was thinking that by using the same value the user would be reminded what the default was that was intentionally removed

Reply:
I  guess this depends on how 'defaulting' works in the UI and how careful it is to distinguish between not-set and set to nothing.  If not-set, I would expect the UI to fill with a default (when a default exists, and the placeholder otherwise).  If set to nothing, I would expect the UI to show nothing (including not showing the placeholder). 

Is placeholder a UI thing in that what is displayed as the placeholder isn't considered an inputted value in any way, whereas default is considered an inputted value?  This is probably a splitting-hairs thing I guess and see that you already committed the update - so moving on.  Just to add, an advantage of the updated approach is that we can more easily determine which of the two values is being used - which might help testability - and perhaps we can then come to a conclusion on what would be best.

Reply:
> Is placeholder a UI thing in that what is displayed as the placeholder isn't considered an inputted value in any way, whereas default is considered an inputted value? 

Yes to both. A placeholder is supposed to provide guidance. Take, for example, an input field for a social security number. `xxx-xxx-xxxx`. It's not a valid entry but makes it clear that the expectation is that the input is supposed to be entered in the specified format. (Placeholders are not used though to validate or enforce anything.) Generally speaking laceholders are a front-end thing only, whereas default applies now to the front-end (is displayed there) and back-end (stored/passed through).